buildSrc/src/main/java/com/amazonaws/blox/swagger/ApiGatewaySecurityFilter.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.swagger;
+
+import io.swagger.models.Operation;
+import io.swagger.models.Path;
+import io.swagger.models.Swagger;
+import io.swagger.models.auth.ApiKeyAuthDefinition;
+import io.swagger.models.auth.In;
+import java.util.Collections;
+import lombok.Getter;
+import lombok.Setter;
+import org.gradle.api.tasks.Input;
+
+@Getter
```

Comment:
I'm a bit wary of using that, since this isn't acutally a POJO/Bean. For example, that will also generate a constructor, `equals`, `hashCode`, and `toString`, which might not make sense for this object.

Do you think that condensing these annotations is worth adding those unused features to this object? If it enhances readability/understandability, then I'd be all for it.